tests/unit/Trace/Connection/RestTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests\Trace\Connection;
```

Comment:
I'm working on moving the unit tests into the `Google\Cloud\Tests\Unit` namespace. Would you mind updating these all to `Google\Cloud\Tests\Unit\Trace` while you're working? :)